src/NuGet.Core/NuGet.Build.Tasks.Pack/PackTask.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.Diagnostics;
+using System.Linq;
```

Comment:
nit: it doesn't appear that this using is needed.

Reply:
yeah good catch.
The namespace was from the older implementation. Fixed.